packages/bpk-component-pagination/src/BpkPagination.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cssModules } from 'bpk-react-utils';
+import BpkPaginationList from './BpkPaginationList';
+import BpkPaginationNudger from './BpkPaginationNudger';
+import STYLES from './bpk-pagination.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkPagination = (props) => {
+  const classNames = [getClassName('bpk-pagination')];
+  const {
+    pageCount,
+    previousLabel,
+    nextLabel,
+    onPageChange,
+    selectedPageIndex,
+    visibleRange,
+    className,
+    ...rest
+  } = props;
+
+  if (className) { classNames.push(getClassName(className)); }
+
+  const handlePageChange = (nextPageIndex) => {
```

Comment:
Minor: We end up creating this function every time `render` is called. Maybe we should instead move it out and change the signature to 

```javascript
const handlePageChange = onPageChange => nextPageIndex => {
  // ...
}
```

This way it can be partially applied in render as `handelPageChange(onPageChange)` and on line 65 we can skip creating an anonymous function extra